hepcrawl/utils.py

```diff
@@ -238,9 +239,9 @@ def range_as_string(data):
     ranges = []
     for key, group in groupby(
         enumerate(data),
-        lambda (index, item): index - item
+            lambda index_item: index_item[0] - index_item[1]
```

Comment:
sad that you can't unpack tuples in `lambda`s in Python 3. what's with the strange indentation though?